src/main/java/uk/gov/hmcts/reform/idam/health/command/ReplicationCommandProbe.java

```diff
@@ -86,102 +119,51 @@ public String getName() {
 
     public String[] getCommand() {
         if (command == null) {
-            command = buildCommand(probeProperties.getCommand().getTemplate(), probeProperties.getCommand().getUser(), probeProperties.getCommand().getPassword());
+            command = buildCommand(probeProperties.getCommand().getTemplate(), probeProperties.getCommand()
+                                           .getHostIdentity() + ".service.core-compute-idam-preview.internal",
```

Comment:
`.service.core-compute-idam-preview.internal` probably needs to be in config?

Reply:
yeah, we just pass the identity in so often already though.

I could run a command to get the hostname, but that also seems a bit unnecessary